src/order/filter/zero_and_bounded_at_filter.lean

```diff
@@ -54,18 +55,18 @@ if `f =O[l] 1`. -/
 def bounded_at_filter [has_norm β] [has_one (α → β)] (l : filter α) (f : α → β) : Prop :=
 asymptotics.is_O l f (1 : α → β)
 
-lemma zero_at_filter_is_bounded_at_filter [normed_field β] (l : filter α) (f : α → β)
+lemma zero_at_filter_is_bounded_at_filter [normed_field β] {l : filter α} {f : α → β}
   (hf : zero_at_filter l f) : bounded_at_filter l f :=
 asymptotics.is_O_of_div_tendsto_nhds (by simp) _ (by { convert hf, ext1, simp, })
 
-lemma zero_is_bounded_at_filter [normed_field β] (l : filter α) :
+lemma zero_is_bounded_at_filter [normed_field β] {l : filter α} :
```

Comment:
What is the motivation for making this argument implicit?

Reply:
The difference with `zero_at_filter_is_bounded_at_filter` is that `l` can be inferred from `hf`. Whereas here there is no way to infer `l`. So I would probably lean towards keeping `l` explicit here.

Reply:
Ah sure I see what you mean. I've fixed this now :)